src/Symfony/Bundle/FrameworkBundle/Tests/Fixtures/Resources/views/translation.html.php

```diff
@@ -46,4 +46,4 @@
 
 <?php echo $view['translator']->trans('typecast', ['a' => (int) '123'], 'not_messages'); ?>
 <?php echo $view['translator']->transChoice('msg1', 10 + 1, [], 'not_messages'); ?>
-<?php echo $view['translator']->transChoice('msg2', intval(4.5), [], 'not_messages'); ?>
+<?php echo $view['translator']->transChoice('msg2', (int)4.5, [], 'not_messages'); ?>
```

Comment:
The `intval()` call was intended here to have a test case with a function call. We could replace it with another function call, but IMO that's not really necessary as we are only talking about a test fixture here.